google/ggrc-core
# GGRC-41 gGRC daily digest email with no indication of how to update email settings / unsubscribe

# Issue description

GGRC daily digest email does not containe indication of how to update email settings / unsubscribe nor it has unsubscribe link

# Steps to test the changes

Generate Daily Digest email or visit `_notifications/show_daily_digest` endpoint. Follow the **unsubscribe** link at the bottom of the email:

If **not** **authorized** - **login dialog** should be displayed, then forwarded to **unsubscribe result page**
If **authorized** - should be forwarded to **unsubscribe result page**

# Solution description

Solution is based on [this](https://github.com/google/ggrc-core/pull/5836) **Pull Request**
Design for unsubscribe Jinja template is [here](https://drive.google.com/corp/drive/folders/1tcUNSGfIJxApRBMdTZujZuqIoMFCeIlL)

- Add `_notifications/unsubscribe` endpoint and **unsubscribe** link to daily digest email body
- Add **unsubscribe** result page
- Add corresponding tests

# Sanity checklist




# PR Review checklist
